src/main/kotlin/fi/metatavu/metaform/server/metaform/FieldTypeMapper.kt

```diff
@@ -0,0 +1,52 @@
+package fi.metatavu.metaform.server.metaform
+
+import fi.metatavu.metaform.api.spec.model.MetaformFieldType
+import org.slf4j.Logger
+import javax.enterprise.context.ApplicationScoped
+import javax.inject.Inject
+
+/**
+ * Mapper for mapping field types into store data types
+ *
+ * @author Antti Leppa
+ */
+@ApplicationScoped
+class FieldTypeMapper {
```

Comment:
Is there a legitimate reason this class is an instance? I am aware that this has been ported from Java but i believe it could be an `object` in kotlin.